### PR TITLE
Improve iOS add-to-app tutorial

### PR DESCRIPTION
**Pull request by loic-sharma**
_Description of what this PR is changing or adding, and why:_ Fix minor issues in the iOS add-to-app tutorial.

_Issues fixed by this PR (if any):_

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/content/add-to-app/ios/add-flutter-screen.md [hunk 1]
```diff
@@ -786,5 +786,5 @@ in any way you'd like, before presenting the Flutter UI using a
 [`SystemNavigator.pop()`]: {{site.api}}/flutter/services/SystemNavigator/pop.html
 [tree-shaken]: https://en.wikipedia.org/wiki/Tree_shaking
 [`WidgetsApp`]: {{site.api}}/flutter/widgets/WidgetsApp-class.html
-[`window.defaultRouteName`]: {{site.api}}/flutter/dart-ui/SingletonFlutterWindow/defaultRouteName.html
+[`PlatformDispatcher.defaultRouteName`]: {{site.api}}/flutter/dart-ui/PlatformDispatcher/defaultRouteName.html
```
Comment (loic-sharma) [inline src/content/add-to-app/ios/add-flutter-screen.md:789]: The `window` global is deprecated: https://docs.flutter.dev/release/breaking-changes/window-singleton

**Review by jmagman — APPROVED** [review]
LGTM, thanks!

**Review by parlough — APPROVED** [review]
Thanks!

**Review by sfshaza2 — APPROVED** [review]
Thanks, @loic-sharma! lgtm

**Pull request merged**